README.md

```diff
@@ -34,6 +34,10 @@ The bq_cred.json is service account for Big Query, it needs to be supplied and p
 `docker cp ~/secrets student_dashboard:/secrets`
 5. Initialize the MySQL database by loading the users and files on the next step. You'll need to be on VPN for this to work.
 
+## Making migration using Django models
+1. creating the file `./manage.py makemigrations dashboard`
+2. applying the changes to DB `./manage.py migrate dashboard`
```

Comment:
To do this locally you actually have to install the requirements, probably with a venv? 

This can also be done by connecting to the docker instance and running it in there, but then you have to copy the file out (also with docker). Maybe we should mention a little more about this since the default path doesn't involve venv.

Reply:
I have added more details as suggested